radio/src/targets/horus/board.cpp

```diff
@@ -175,6 +175,7 @@ void boardInit()
   init2MhzTimer();
   init1msTimer();
   usbInit();
+  usbDeInit();
```

Comment:
Would you add a small explanation in 'next' directly? (why Init followed directly by DeInit)
I will test your change now on X7
Thanks!